packages/vitest/LICENSE.md

```diff
@@ -197,6 +197,35 @@ Repository: https://github.com/sinonjs/fake-timers.git
 
 ---------------------------------------
 
+## acorn
```

Comment:
Why do we bundle acorn now?

Reply:
<img width="693" alt="image" src="https://github.com/vitest-dev/vitest/assets/29533304/1d94beea-254c-4b8b-bc1f-80255d0d838e">

Since we removed `acorn` from the dependencies and now `external` no longer contains `acorn`, we now need to explicitly write `acorn` to `external`